Error - 'module' object is not callable
Reason- Both module and fuction bear the same name with causes a conflict.
Solution - 
Import fuction as- import fizzbuzz as fb
Call function as-     fb.fizzbuzz(100)